### PR TITLE
chore: update Grid Live demo

### PR DESCRIPTION
**Pull request by mbechev**
- Move the useful button links as discussed in this issue https://github.com/telerik/k2/issues/406.
- Remove the unnecessary horizontal scrollbar, and replace the **style** property from the codebase
- Minor CSS updates in other elements.

### CODE REVIEW

File: examples-standalone/grid-live-data/src/app/main-panel/transactions/transactions.component.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
                 height="40px"
                 [imageSrc]="getCardImg(card)"
                 shape="circle"
-                style="background-color: transparent"
+                [style.background-color]="'transparent'"
```
Comment (Raisolution) [inline examples-standalone/grid-live-data/src/app/main-panel/transactions/transactions.component.html:10]: what is wrong with the previous line? + the syntax of the new one is incorrect :)

File: examples-standalone/grid-live-data/src/app/header/header.component.html [hunk 2]
```diff
@@ -6,13 +6,33 @@ <h1 class="title">CryptoVault</h1>
     <kendo-appbar-spacer></kendo-appbar-spacer>
 
     <kendo-appbar-section class="actions">
+        <kendo-badge-container>
```
Comment (Raisolution) [inline examples-standalone/grid-live-data/src/app/header/header.component.html:9]: what is the purpose of the badge container here? We don't have badges in it.

File: examples-standalone/grid-live-data/src/app/my-portfolio/my-portfolio.component.html [hunk 3]
```diff
@@ -7,7 +7,7 @@
                 height="40px"
                 [imageSrc]="getImg(card)"
                 shape="circle"
-                style="background-color: transparent"
+                [style.background-color]="'transparent'"
```
Comment (Raisolution) [inline examples-standalone/grid-live-data/src/app/my-portfolio/my-portfolio.component.html:10]: ditto

**Review by Raisolution — CHANGES_REQUESTED** [review]

File: examples-standalone/grid-live-data/src/app/main-panel/transactions/transactions.component.html (hunk 1 above)
Comment (mbechev) [reply examples-standalone/grid-live-data/src/app/main-panel/transactions/transactions.component.html:10]: I thought that the **style** property in HTML needs to be replaced with **class** instead. That is why I decided to not make a separate class for a single CSS property, and I just replace it with the current syntax. If need I could revert it.

**Review by Raisolution — APPROVED** [review]

**Pull request merged**